### PR TITLE
Add API controller for Job Templates

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: app/controllers/api/v2/job_templates_controller.rb [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      end
+
+      private
+
+      def resource_class
+        JobTemplate
+      end
+
+      def allowed_nested_id
+        %w(operatingsystem_id location_id organization_id)
```
Comment (ohadlevy) [inline app/controllers/api/v2/job_templates_controller.rb]: do we nest by operatingsystem ?

**Comment by iNecas** [comment]
ACK, tested and works nicely

**Comment by stbenjam** [comment]
Thanks @iNecas! How do we work in this repo? (If it's Katello-style I have no rights to merge my PR.)

**Comment by iNecas** [comment]
Merging for you, will get you merge rights on this repo

**Pull request merged**